server/resources/User.py

```diff
@@ -72,7 +72,7 @@ def put(cls, user_id: int):
         if user and user.id == current_user:
             try:
                 #loading in form data through validation/clean by marshmallow
-                user.username = user_json["username"]
+                # user.username = user_json["username"]
```

Comment:
was this supposed to be deleted? if so remove comment

Reply:
Future state of the page should allow the user to edit their username, can remove from this iteration